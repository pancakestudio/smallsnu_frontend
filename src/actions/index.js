import * as types from './ActionTypes';

export function buildingClick(e){
  return{
    type: types.BUILDING_CLICK,
    e
  };
}
