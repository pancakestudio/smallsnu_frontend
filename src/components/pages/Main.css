.mainPage {
  display: block;
  position: relative;
  z-index: 0;
  height: 100%;
  transition: 0.3s;
}
