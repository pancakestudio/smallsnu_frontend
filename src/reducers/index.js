import { combineReducers } from 'redux';
import buildingButton from './buildingButton';

const reducers = combineReducers({
  buildingButton
});

export default reducers;
