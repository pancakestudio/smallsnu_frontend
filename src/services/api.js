import axios from 'axios'

axios.defaults.baseURL = 'http://127.0.0.1:8000/'

export function getBuildingInfo(bldgNo){
  return axios.get(`building/${bldgNo}`)
    .catch(error => { return {error: error}})
}

export function getRestaurantInfo(){
  return axios.get(`restaurant/`)
    .catch(error => {return {error: error}})
}

export function getSeminarInfo(){
  return axios.get('seminar/')
    .catch(error => { return {error:error}})
}
