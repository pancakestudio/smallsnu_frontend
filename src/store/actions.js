import * as types from './actionTypes'

/******** Map ********/

export function requestBuilding(bldgNo){
  return{
    type: types.REQUEST_BUILDING,
    bldgNo
  };
}

export function getBuildingSuccess(krName, bldgNo, info, restaurants, seminars, lectures, posts){
  return{
    type: types.GET_BUILDING_SUCCESS,
    krName,
    bldgNo,
    info,
    restaurants,
    seminars,
    lectures,
    posts,
  }
}

export function getBuildingFailure(error){
  return{
    type: types.GET_BUILDING_FAILURE,
    error
  }
}

export function zoomChanged(zoomLevel){
  return {
    type: types.ZOOM_CHANGED,
    zoomLevel
  }
}

export function hideMarkers(){
  return {
    type: types.HIDE_MARKERS
  }
}

/******** SideBar ********/

export function toggleSidebar(){
  return {
    type: types.TOGGLE_SIDE_BAR
  }
}

export function toggleShuttleMenuMarker(){
  return{
    type: types.TOGGLE_SHUTTLE_MENU_MARKER
  }
}

/******** SearchBar ********/

export function search(bldgNo, bldgPos){
  return{
    type: types.SEARCH,
    bldgNo,
    bldgPos
  }
}

/******** Shuttle ********/
export function requestAllShuttles(){
  return{
    type: types.REQUEST_ALL_SHUTTLE
  }
}

export function getAllShuttleSuccess(stations){
  return{
    type: types.GET_ALL_SHUTTLE_SUCCESS,
    stations
  }
}

export function toggleShuttleMarker(){
  return{
    type: types.TOGGLE_SHUTTLE_MARKER
  }
}

export function requestAllRevShuttles(){
  return{
    type: types.REQUEST_ALL_REV_SHUTTLE
  }
}

export function getAllRevShuttleSuccess(stations){
  return{
    type: types.GET_ALL_REV_SHUTTLE_SUCCESS,
    stations
  }
}

export function toggleRevShuttleMarker(){
  return{
    type: types.TOGGLE_REV_SHUTTLE_MARKER
  }
}

/******** Post ********/

export function requestBoard(bldgNo){
  return {
    type: types.REQUEST_BOARD,
    bldgNo
  }
}

export function getBoardSuccess(posts, bldgNo){
  return {
    type: types.GET_BOARD_SUCCESS,
    posts,
    bldgNo
  }
}

export function getBoardFailure(error){
  return {
    type: types.GET_BOARD_FAILURE,
    error
  }
}

export function requestPost(id) {
  return {
    type: types.REQUEST_POST,
    id
  }
}

export function getPostSuccess(post){
  return {
    type: types.GET_POST_SUCCESS,
    post
  }
}

export function getPostFailure(error){
  return {
    type: types.GET_POST_FAILURE,
    error
  }
}

export function savePost(post, bldgNo){
  return{
    type: types.SAVE_POST,
    post,
    bldgNo
  }
}

export function savePostSuccess(bldgNo){
  return {
    type: types.SAVE_POST_SUCCESS,
    bldgNo
  }
}

export function savePostFailure(error){
  return {
    type: types.SAVE_POST_FAILURE,
    error
  }
}

export function editPost(post, bldgNo){
  return{
    type: types.EDIT_POST,
    post,
    bldgNo,
  }
}

export function editPostSuccess(postId){
  return {
    type: types.EDIT_POST_SUCCESS,
    postId
  }
}

export function editPostFailure(error){
  return {
    type: types.EDIT_POST_FAILURE,
    error
  }
}

export function editPostFlag(){
  return{
    type: types.EDIT_POST_FLAG,
  }
}

export function deletePost(post, bldgNo, password){
  return{
    type: types.DELETE_POST,
    post,
    bldgNo,
    password
  }
}

export function deletePostSuccess(bldgNo){
  return {
    type: types.DELETE_POST_SUCCESS,
    bldgNo
  }
}

export function deletePostFailure(error){
  return {
    type: types.DELETE_POST_FAILURE,
    error
  }
}

export function showWritePost(bldgNo) {
  return{
    type: types.SHOW_WRITE_POST,
    bldgNo
  };
}

export function hideWritePost() {
  return {
    type: types.HIDE_WRITE_POST,
  }
}

export function showPasswordCheck(target){
  return{
    type: types.SHOW_PASSWORD_CHECK,
    target
  }
}

export function hidePasswordCheck(){
  return{
    type: types.HIDE_PASSWORD_CHECK
  }
}

export function postLike(postId) {
  return {
    type: types.POST_LIKE,
    postId
  }
}

export function postLikeSuccess(postId) {
  return {
    type: types.POST_LIKE_SUCCESS,
    postId
  }
}

export function postLikeFailure(error) {
  return {
    type: types.POST_LIKE_FAILURE,
    error
  }
}

export function saveComment(comment, postId){
  return {
    type: types.SAVE_COMMENT,
    comment,
    postId
  }
}

export function saveCommentSuccess(postId){
  return {
    type: types.SAVE_COMMENT_SUCCESS,
    postId
  }
}

export function saveCommentFailure(error){
  return {
    type: types.SAVE_COMMENT_FAILURE,
    error
  }
}

export function showEditComment(comment){
  return {
    type: types.SHOW_EDIT_COMMENT,
    comment
  }
}

export function hideEditComment(){
  return {
    type: types.HIDE_EDIT_COMMENT
  }
}

export function editComment(comment, postId){
  return {
    type: types.EDIT_COMMENT,
    comment,
    postId
  }
}

export function editCommentSuccess(postId){
  return {
    type: types.EDIT_COMMENT_SUCCESS,
    postId
  }
}

export function editCommentFailure(error){
  return {
    type: types.EDIT_COMMENT_FAILURE,
    error
  }
}

export function deleteComment(comment, postId, password){
  return {
    type: types.DELETE_COMMENT,
    comment,
    postId,
    password
  }
}

export function deleteCommentSuccess(msg){
  return {
    type: types.DELETE_COMMENT_SUCCESS,
    msg
  }
}

export function deleteCommentFailure(error){
  return {
    type: types.DELETE_COMMENT_FAILURE,
    error
  }
}

export function commentLike(commentId, postId) {
  return {
    type: types.COMMENT_LIKE,
    commentId,
    postId
  }
}

export function commentLikeSuccess(postId) {
  return {
    type: types.COMMENT_LIKE_SUCCESS,
    postId
  }
}

export function commentLikeFailure(error) {
  return {
    type: types.COMMENT_LIKE_FAILURE,
    error
  }
}

export function changeBoardPage(page){
  return {
    type: types.CHANGE_BOARD_PAGE,
    page
  }
}

export function wrongPassword(){
  return {
    type: types.WRONG_PASSWORD,
  }
}

/******** Restaurant ********/

export function toggleResMarker(){
  return{
    type: types.TOGGLE_RES_MARKER,
  }
}

export function requestRestaurant(id){
  return{
    type: types.REQUEST_RESTAURANT,
    id
  }
}

export function getRestaurantSuccess(restaurant){
  return{
    type : types.GET_RESTAURANT_SUCCESS,
    restaurant,
  }
}

export function getRestaurantFailure(error){
  return{
    type: types.GET_RESTAURANT_FAILURE,
    error
  }
}

export function requestAllRestaurants(){
  return{
    type: types.REQUEST_ALL_RESTAURANTS,
  }
}


export function getAllRestaurantsSuccess(restaurants){
  return{
    type : types.GET_ALL_RESTAURANTS_SUCCESS,
    restaurants,
  }
}

export function getAllRestaurantsFailure(error){
  return{
    type: types.GET_ALL_RESTAURANTS_FAILURE,
    error
  }
}

/******** Seminar ********/

export function toggleSemiMarker(){
  return{
    type: types.TOGGLE_SEMI_MARKER,
  }
}

export function changeSeminarPage(page){
  return {
    type: types.CHANGE_SEMINAR_PAGE,
    page
  }
}

export function requestSeminar(id){
  return {
    type: types.REQUEST_SEMINAR,
    id
  }
}

export function getSeminarSuccess(seminar){
  return {
    type: types.GET_SEMINAR_SUCCESS,
    seminar
  }
}

export function getSeminarFailure(error){
  return {
    type: types.GET_SEMINAR_FAILURE,
    error
  }
}

export function requestBldgSeminars(bldgNo){
  return {
    type: types.REQUEST_BLDG_SEMINARS,
    bldgNo
  }
}

export function getBldgSeminarsSuccess(seminars, bldgNo){
  return {
    type: types.GET_BLDG_SEMINARS_SUCCESS,
    seminars,
    bldgNo
  }
}

export function getBldgSeminarsFailure(error){
  return {
    type: types.GET_BLDG_SEMINARS_FAILURE,
    error
  }
}

export function requestAllSeminars(){
  return {
    type: types.REQUEST_ALL_SEMINARS,
  }
}

export function getAllSeminarsSuccess(seminars){
  return {
    type: types.GET_ALL_SEMINARS_SUCCESS,
    seminars
  }
}

export function getAllSeminarsFailure(error){
  return {
    type: types.GET_ALL_SEMINARS_FAILURE,
    error
  }
}

/******** Bank ********/

export function toggleBankMarker(){
  return{
    type: types.TOGGLE_BANK_MARKER,
  }
}

export function requestAllBanks(){
  return{
    type: types.REQUEST_ALL_BANKS,
  }
}

export function getAllBanksSuccess(banks){
  return{
    type: types.GET_ALL_BANKS_SUCCESS,
    banks,
  }
}

export function getAllBanksFailure(error){
  return{
    type: types.GET_ALL_BANKS_FAILURE,
    error,
  }
}

export function requestBank(id){
  return{
    type: types.REQUEST_BANK,
    id
  }
}

export function getBankSuccess(bank){
  return{
    type: types.GET_BANK_SUCCESS,
    bank
  }
}

export function getBankFailure(error){
  return{
    type: types.GET_BANK_FAILURE,
    error
  }
}


/******** ATM ********/

export function toggleATMMarker(){
  return{
    type: types.TOGGLE_ATM_MARKER,
  }
}

export function requestAllATMs(){
  return{
    type: types.REQUEST_ALL_ATMS,
  }
}

export function getAllATMsSuccess(atms){
  return{
    type: types.GET_ALL_ATMS_SUCCESS,
    atms,
  }
}

export function getAllATMsFailure(error){
  return{
    type: types.GET_ALL_ATMS_FAILURE,
    error,
  }
}

export function requestATM(id){
  return{
    type: types.REQUEST_ATM,
    id
  }
}

export function getATMSuccess(atm){
  return{
    type: types.GET_ATM_SUCCESS,
    atm
  }
}

export function getATMFailure(error){
  return{
    type: types.GET_ATM_FAILURE,
    error
  }
}

/******** Cafe ********/

export function toggleCafeMarker(){
  return{
    type: types.TOGGLE_CAFE_MARKER,
  }
}

export function requestAllCafes(){
  return{
    type: types.REQUEST_ALL_CAFES,
  }
}

export function getAllCafesSuccess(cafes){
  return{
    type: types.GET_ALL_CAFES_SUCCESS,
    cafes,
  }
}

export function getAllCafesFailure(error){
  return{
    type: types.GET_ALL_CAFES_FAILURE,
    error,
  }
}

export function requestCafe(id){
  return{
    type: types.REQUEST_CAFE,
    id
  }
}

export function getCafeSuccess(cafe){
  return{
    type: types.GET_CAFE_SUCCESS,
    cafe
  }
}

export function getCafeFailure(error){
  return{
    type: types.GET_CAFE_FAILURE,
    error
  }
}

/******** Conv ********/

export function toggleConvMarker(){
  return{
    type: types.TOGGLE_CONV_MARKER,
  }
}

export function requestAllConves(){
  return{
    type: types.REQUEST_ALL_CONVES,
  }
}

export function getAllConvesSuccess(conves){
  return{
    type: types.GET_ALL_CONVES_SUCCESS,
    conves,
  }
}

export function getAllConvesFailure(error){
  return{
    type: types.GET_ALL_CONVES_FAILURE,
    error,
  }
}

export function requestConv(id){
  return{
    type: types.REQUEST_CONV,
    id
  }
}

export function getConvSuccess(conv){
  return{
    type: types.GET_CONV_SUCCESS,
    conv
  }
}

export function getConvFailure(error){
  return{
    type: types.GET_CONV_FAILURE,
    error
  }
}


/******** Main ********/

export function refreshData(){
  return {
    type: types.REFRESH_DATA
  }
}
