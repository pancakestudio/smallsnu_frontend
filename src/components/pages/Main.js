import React from 'react'
import Map from '../../containers/Map'
import { Header } from '../../components/organisms/Header'
import BuildingModal from '../../containers/BuildingModal'
import RestaurantModal from '../../containers/RestaurantModal'
import SeminarModal from '../../containers/SeminarModal'
import SeminarListModal from '../../containers/SeminarListModal'
import Sidebar from '../../containers/Sidebar'
import './Main.css'

const Main = (props) => {
  return (
    <div className = "mainPage">
      <Sidebar />
      <Header />
      <Map />
      <BuildingModal />
      <RestaurantModal />
      <SeminarModal />
      <SeminarListModal />
    </div>
  )
}

export default Main
