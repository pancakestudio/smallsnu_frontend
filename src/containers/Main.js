import { connect } from 'react-redux'
import { Main } from '../components/pages/Main'
import {
  requestBuilding,
  requestBoard, requestPost,
  requestRestaurant,
  requestSeminar, requestBldgSeminars,
  requestATM, requestBank, requestCafe, requestConv
} from '../store/actions'

const mapStateToProps = (state) => {
  return {
    error: state.error
  }
}

const mapDispatchToProps = (dispatch) => {
  return {
    getBuilding: (id) => {
      dispatch(requestBuilding(id))
    },
    getRestaurant: (id) => {
      dispatch(requestRestaurant(id))
    },
    getATM: (id) => {
      dispatch(requestATM(id))
    },
    getBank: (id) => {
      dispatch(requestBank(id))
    },
    getCafe: (id) => {
      dispatch(requestCafe(id))
    },
    getConv: (id) => {
      dispatch(requestConv(id))
    },
    getSeminar: (id) => {
      dispatch(requestSeminar(id))
    },
    getSeminarList: (bldgNo) => {
      dispatch(requestBldgSeminars(bldgNo))
    },
    getBoard: (id) => {
      dispatch(requestBoard(id))
    },
    getPost: (id) => {
      dispatch(requestPost(id))
    }
  }
}

export default connect(mapStateToProps, mapDispatchToProps)(Main)
