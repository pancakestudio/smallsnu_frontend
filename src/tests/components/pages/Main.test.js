import React from 'react';
import createSagaMiddleware from 'redux-saga'
import configureStore from 'redux-mock-store'
import { shallow, mount } from 'enzyme'
import { applyMiddleware, createStore } from 'redux'
import { Provider } from 'react-redux'
import { Main } from '../../../components/pages/Main'
import ConnectedMain from '../../../containers/Main'
import * as actions from '../../../store/actions'
import rootSaga from '../../../store/sagas'
import * as api from '../../../services/api'
import reducers from '../../../store/reducers'
import Router from '../../../BrowserRouter'

describe('Main', ()=>{
  const mockGetBuilding = jest.fn()
  const mockGetRestaurant = jest.fn()
  const mockGetSeminar = jest.fn()
  const mockGetSeminarList = jest.fn()
  const mockGetBoard = jest.fn()
  const mockGetPost = jest.fn()
  let component
  it('renders correctly', ()=>{
    component = shallow(
      <Main
        match={{params: {}}}
        error={""}
        getBuilding={mockGetBuilding}
        getRestaurant={mockGetRestaurant}
        getSeminar={mockGetSeminar}
        getSeminarList={mockGetSeminarList}
        getBoard={mockGetBoard}
        getPost={mockGetPost}
      />
    )
  })

  it('matches snapshot', ()=>{
    expect(component).toMatchSnapshot()
  })

  it('calls functions', ()=>{
    component.setProps({match: {params: {modal: "building", id: "62"}}})
    expect(mockGetBuilding.mock.calls.length).toBe(1)
    component.setProps({match: {params: {modal: "restaurant", id: "2"}}})
    expect(mockGetRestaurant.mock.calls.length).toBe(1)
    component.setProps({match: {params: {modal: "seminar", id: "3"}}})
    expect(mockGetSeminar.mock.calls.length).toBe(1)
    component.setProps({match: {params: {modal: "seminarlist", id: "301"}}})
    expect(mockGetSeminarList.mock.calls.length).toBe(1)
    component.setProps({match: {params: {modal: "board", id: "302"}}})
    expect(mockGetBoard.mock.calls.length).toBe(1)
    component.setProps({match: {params: {modal: "post", id: "4"}}})
    expect(mockGetPost.mock.calls.length).toBe(1)
  })

})

describe('ConnectedMain', ()=>{
  const buildingRequest = jest
    .spyOn(api, 'getBuildingInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: {
        kr_name: "중앙도서관",
        code: "62",
        info: "hello, world!",
        restaurants: ["rests"],
        seminars: ["seminars"],
        lectures: ["lecs"],
        posts: ["posts"]
      }
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  const restaurantRequest = jest
    .spyOn(api, 'getRestaurantInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: {
        id: "1",
        info: "hello, world!",
      }
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  const seminarRequest = jest
    .spyOn(api, 'getSeminarInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: {
        id: "2",
        info: "hello, world!",
      }
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  const seminarListRequest = jest
    .spyOn(api, 'getBldgSeminarInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: [
        {
          id: "3",
          info: "hello, world!",
        }
      ]
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  const boardRequest = jest
    .spyOn(api, 'getBoardInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: [
        {
          id: "4",
          info: "hello, world!",
        }
      ]
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  const postRequest = jest
    .spyOn(api, 'getPostInfo')
    .mockImplementationOnce(()=>Promise.resolve({
      data: {
        id: "5",
        info: "hello, world!",
      }
    }))
    .mockImplementationOnce(()=>Promise.resolve({
      error: "error"
    }))
  let store, component

  it('renders correctly', ()=>{
    const sagaMiddleware = createSagaMiddleware()
    const mockStore = configureStore([sagaMiddleware])
    store = mockStore()
    sagaMiddleware.run(rootSaga)
    component = mount(
      <Router>
        <Provider store={store}>
          <ConnectedMain
            match={{params: {modal: "", id: ""}}}
          />
        </Provider>
      </Router>)
  })

  it('matches snapshot', ()=>{
    expect(component).toMatchSnapshot()
  })

  it('reducers', async ()=>{
    const sagaMiddleware = createSagaMiddleware()

    store = createStore(reducers, applyMiddleware(sagaMiddleware))
    sagaMiddleware.run(rootSaga)

    store.dispatch(actions.requestBuilding('62'))
    await Promise.resolve()
    expect(store.getState().selectedBldg.krName).toBe('중앙도서관')
    expect(store.getState().selectedBldg.bldgNo).toBe('62')
    expect(store.getState().selectedBldg.info).toBe('hello, world!')
    expect(store.getState().selectedBldg.restaurants).toEqual(['rests'])
    expect(store.getState().selectedBldg.seminars).toEqual(['seminars'])
    expect(store.getState().selectedBldg.posts).toEqual(['posts'])

    store.dispatch(actions.requestBuilding(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('건물 정보를 받아오지 못했습니다.')

    store.dispatch(actions.requestRestaurant('1'))
    await Promise.resolve()
    expect(store.getState().selectedRes.id).toBe('1')
    expect(store.getState().selectedRes.info).toBe('hello, world!')

    store.dispatch(actions.requestRestaurant(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('식당 정보를 받아오지 못했습니다.')

    store.dispatch(actions.requestSeminar('2'))
    await Promise.resolve()
    expect(store.getState().selectedSemi.id).toBe('2')
    expect(store.getState().selectedSemi.info).toBe('hello, world!')

    store.dispatch(actions.requestSeminar(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('세미나 정보를 받아오지 못했습니다.')

    store.dispatch(actions.requestBldgSeminars('301'))
    await Promise.resolve()
    expect(store.getState().selectedSemiList[0].id).toBe('3')
    expect(store.getState().selectedSemiList[0].info).toBe('hello, world!')

    store.dispatch(actions.requestBldgSeminars(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('세미나 정보를 받아오지 못했습니다.')

    store.dispatch(actions.requestBoard('300'))
    await Promise.resolve()
    expect(store.getState().selectedPostList[0].id).toBe('4')
    expect(store.getState().selectedPostList[0].info).toBe('hello, world!')

    store.dispatch(actions.requestBoard(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('게시판 정보를 받아오지 못했습니다.')

    store.dispatch(actions.requestPost('5'))
    await Promise.resolve()
    expect(store.getState().selectedPost.id).toBe('5')
    expect(store.getState().selectedPost.info).toBe('hello, world!')

    store.dispatch(actions.requestPost(0))
    await Promise.resolve()
    expect(store.getState().error).toBe('게시글 정보를 받아오지 못했습니다.')
  })
})
