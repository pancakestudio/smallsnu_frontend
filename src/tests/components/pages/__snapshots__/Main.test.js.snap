// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectedMain matches snapshot 1`] = `undefined`;

exports[`Main matches snapshot 1`] = `undefined`;
